go.mod

```
@@ -2,8 +2,8 @@ module github.com/jfrog/jfrog-client-go
 
 require (
 	github.com/buger/jsonparser v1.1.1
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/go-git/go-git/v5 v5.4.2
+	github.com/golang-jwt/jwt/v4 v4.0.0
```

```suggestion
	github.com/golang-jwt/jwt/v4 v4.1.0
```

Let's upgrade to 4.1.0 to make it work on Go 1.17 that we recently started to work with:
https://github.com/golang-jwt/jwt/releases/tag/v4.1.0